draft-packages/menu/KaizenDraft/Menu/components/MenuContent.module.scss

```diff
@@ -50,16 +49,16 @@ $gutter: 4px;
   border: none;
   text-align: left;
   padding: 8px $side-padding;
-  margin: 0 $gutter;
+  margin: 0 $gutter + 2px;
```

Comment:
It's a bit weird having `$gutter` be 4px but two of the three times it's used we want 6px.

Could we use two variables (one 4px and one 6px) with better names? `contentGutter` and `headerGutter`?